src/target/adiv5.h

```diff
@@ -89,23 +89,19 @@
 #define ADIV5_AP_BASE		ADIV5_AP_REG(0xF8)
 #define ADIV5_AP_IDR		ADIV5_AP_REG(0xFC)
 
-/* Known designers seen in SYSROM-PIDR and JTAG IDCode.
- * Ignore Bit 0 from the designer bits to get JEDEC Ids.
- * Should get it's one file as not only related to Adiv5!
- */
+/* Known designers seen in SYSROM-PIDR. Ignore Bit 0 from
+ * the designer bits to get JEDEC Ids with bit 7 ignored.*/
 #define AP_DESIGNER_FREESCALE    0x00e
 #define AP_DESIGNER_TEXAS        0x017
 #define AP_DESIGNER_ATMEL        0x01f
 #define AP_DESIGNER_STM          0x020
 #define AP_DESIGNER_CYPRESS      0x034
 #define AP_DESIGNER_INFINEON     0x041
-#define DESIGNER_XILINX          0x049
```

Comment:
Why are we loosing these defines? (It is missing the `AP_` prefix but it might be useful in the future. Unless it is wrong.

Reply:
As discussed, this is being added back in through the upcoming scanchain-xilinx branch.